source/initial_temperature/S40RTS_perturbation.cc

```diff
@@ -194,17 +194,17 @@ namespace aspect
       const unsigned int num_spline_knots = 21;
 
       // get the spherical harmonics coefficients
-      const std::vector<double> a_lm = spherical_harmonics_lookup->cos_coeffs();
-      const std::vector<double> b_lm = spherical_harmonics_lookup->sin_coeffs();
+      const std::vector<double> a_lm & = spherical_harmonics_lookup->cos_coeffs();
```

Comment:
Nice find. I am more used to `int &a = ...` than `int a & = ...`, is there any difference in meaning? The tester doesnt seem to like it either.

Reply:
Oops. That is me not compiling things. :-)